lib/portage/package/ebuild/config.py

```diff
@@ -3446,7 +3446,7 @@ def archlist(self):
     def selinux_enabled(self):
         if getattr(self, "_selinux_enabled", None) is None:
             self._selinux_enabled = 0
-            if "selinux" in self["USE"].split():
+            if "selinux" in self["USE"].split() and "selinux" in self.features:
```

Comment:
I don't really see why we are checking USE at all here. I think we should check FEATURES exclusively.

In trying to track this down, I found eb2b07ef27bc8f790e2ab280bbd19a012841f9cf which seems relevant.

Reply:
I don't really see a reason to check USE either here, I assumed there was a reason for it, but looking at that commit doesn't seem like it's necessary (or desired). Also worth noting that `selinux` is missing from `make.conf(5)` under the `FEATURES` section, so we should probably add that in. However as this is a fairly big change, cc'ing @perfinion and @0xC0ncord .